_posts/samus_returns/2024-10-25-decrypted-copy.md

```diff
@@ -0,0 +1,64 @@
+---
+layout: post
+title: "How to dump a decrypted copy of Metroid: Samus Returns"
+tags: samus_returns
+---
+Using Godmode9 from your 3DS, you can dump a decrypted copy of the game. The end result will provide you with one of the following files: .3ds, .cia, .app, or .cxi.
+
+## Game Cartridge
+
+If you're using the NTSC version, navigate to **[C:] GAMECART -> 00040000001BB200_v00.trim.3ds**.
+
+If you're using the PAL version, navigate to **[C:] GAMECART -> 00040000001BFB00_v00.trim.3ds**.
+
+{% include blog/figure.html src="/assets/guides/godmode9/godmode9-cart-1.png" caption="Godmode9 screen for game cartridge" %}
+
+### 3DS File (Recommended)
+
+Navigate to **NCSD image options... -> Decrypt file (0:/gm9/out)**, then press A to confirm. 
+
+Copy the .3ds file from this directory on the SD Card to some place on your computer.
+
+{% include blog/figure.html src="/assets/guides/godmode9/godmode9-cart-2.png" caption="Menu to decrypt 3DS file to output folder on SD Card" %}
+
+### CIA File
```

Comment:
if this is not recommended, what else does this provide? less space?

Reply:
I'm just providing all the options. I think it's good to have.